README.md

```diff
@@ -33,6 +33,14 @@ Consider a 30 minute flush interval (1800 seconds)
 - Polygon: 2 second blocks = (1800 / 2) = `900 blocks`
 - Arbitrum: 0.26 second blocks = (1800 / 0.26) = `~6950 blocks`
 
+### Flush Only Mode
+
+This relayer also supports a `--flush-only-mode`. This mode will only flush the chain and not actively listen for new events as they occur. This is useful for running a secondary relayer which "lags" behind the primary relayer. It is only responsible for retrying failed transactions. 
+
```

Comment:
This would require 2 relayer binaries to be running infra side yea?

Could this be condensed into a background go routine, which when toggled, goes back to the `3*` and `4*loopback` flushes?

Reply:
Yes that is correct. The original plan was to have dYdX run this relayer. The hopes is this relayer will pick up any missed transactions if the main relayer goes offline.